rust_src/remacs-macros/lib.rs

```diff
@@ -129,14 +130,8 @@ pub fn lisp_fn(attr_ts: TokenStream, fn_ts: TokenStream) -> TokenStream {
         }
     };
 
-    // we could put #fn_item into the quoted code above, but doing so
-    // drops all of the line numbers on the floor and causes the
-    // compiler to attribute any errors in the function to the macro
-    // invocation instead.
-    // Note: TokenStream has a FromIterator trait impl that converts
-    // an iterator over Token{Stream,Tree,Node}s into a single
-    // TokenStream; collect() calls that impl for us.
-    vec![tokens.parse().unwrap(), fn_ts].into_iter().collect()
+    tokens.append_all(fn_item.into_tokens());
```

Comment:
Please leave a comment telling future readers why the `fn_item` tokens are being appended.

Reply:
Thanks for catching that, you're absolutely correct, of course.

Reply:
No problem. I dropped my -1. Now we wait for @db48x to add their +1